utils/schema.py

```diff
@@ -0,0 +1,21 @@
+import graphene
+
+
+class CountConnection(graphene.Connection):
+    class Meta:
+        abstract = True
+
+    count = graphene.Int(
+        description="Count of nodes on this connection with filters applied",
+        required=True,
+    )
+    total_count = graphene.Int(
+        description="Total count of nodes on this connection regardless of filters",
+        required=True,
+    )
+
+    def resolve_count(self, info):
+        return self.iterable.distinct().count()
+
+    def resolve_total_count(self, info, **kwargs):
+        return self.iterable.model.objects.distinct().count()
```

Comment:
does it need to be `distinct` here? since it just implies `objects.count()` I don;t think it should have any duplicates. 🤔 or did you get duplicates here?

Reply:
The tests seem to work fine without the `distinct`, so we can get rid of this. 👍